config/environments/development.rb

```diff
@@ -5,6 +5,10 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Raise when request includes unpermitted params to help find errors in
+  # dev/test (default value is `nil`, which is used in production)
+  config.action_controller.action_on_unpermitted_parameters = :raise
+
```

Comment:
I don't believe we want this on in production. It should suffice to turn it on in dev and test.